darkhttpd.c

```diff
@@ -2044,6 +2052,13 @@ static void generate_dir_listing(struct connection *conn, const char *path,
             append(listing, "/\n");
         else {
             appendl(listing, spaces, maxlen-strlen(list[i]->name));
+            append(listing, " ");
+            char buf[DIR_LIST_MTIME_SIZE];
+            strftime(buf,
+                     sizeof buf,
+                     DIR_LIST_MTIME_FORMAT,
+                     localtime(&list[i]->mtime.tv_sec));
+            append(listing, buf);
             appendf(listing, "%10llu\n", llu(list[i]->size));
```

Comment:
Should there be an extra space? `" %10llu\n"` ?

Reply:
That is the issue on files that are 1GB :-). I will add that.

What happens, is it formats with spaces automatically, I guess up to however many digits.